src/App.tsx

```diff
@@ -17,18 +48,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={setQuery}
+                status={status}
```

Comment:
Ensure that the value from `event.target.value` is a valid `Filters` enum before setting the state. This can prevent potential runtime errors if the value is not one of the expected enum values.